UPGRADE-2.0.md

```diff
@@ -1,17 +1,92 @@
 # UPGRADE FROM `1.14` TO `2.0`
 
+## Initial part
+
+Even if your app is barely customized, it will require some manual adjustments before it can run again. Depending on
+whether you use Symfony Flex, some of these changes may be applied automatically, but it’s important to check them
+manually regardless.
+
+```md
+#config/packages/_sylius.yaml
+
+imports:
+...
++   - { resource: "@SyliusPayumBundle/Resources/config/app/config.yaml" }
+```
+
+```md
+#config/routes/sylius_shop.yaml
+
+sylius_shop_payum:
+-   resource: "@SyliusShopBundle/Resources/config/routing/payum.yml"
++   resource: "@SyliusPayumBundle/Resources/config/routing/integrations/sylius_shop.yaml"
+
+sylius_payment_notify:
++   resource: "@SyliusPaymentBundle/Resources/config/routing/integrations/sylius.yaml"
```

Comment:
I’m not sure whether it should be configured in the `config/routes/sylius_shop.yaml` shop file or in a separate one.

Reply:
Can be in the `sylius_shop` routing file, **but** should not be localised (since that'd make the route unnecessarily stateful), it might be a good idea to add this info here (same deal w/ payum routes).